webapp/src/shared/context/auth-context.tsx

```diff
@@ -80,6 +98,44 @@ export const AuthProvider = ({ children }: { children: ReactNode }) => {
     [isLoading, authUser, logout],
   );
 
+  useEffect(() => {
```

Comment:
This is mainly why do we need this PR. If we are admins, we may enable/disable endpoints by assigning roles to a user, but we need a way to update in real time other user's authUser when we do so. This is my solution.